sklearn/linear_model/tests/test_base.py

```diff
@@ -233,8 +235,9 @@ def test_linear_regression_sparse_equal_dense(normalize, fit_intercept):
     assert_allclose(clf_dense.coef_, clf_sparse.coef_)
 
 
-def test_linear_regression_multiple_outcome(random_state=0):
+def test_linear_regression_multiple_outcome(global_random_seed):
```

Comment:
We can set a local random state here.
We only check that we fit idependently on different target and the randomness will not impact anything apart of creating the dataset.

Reply:
Good point, addressed.